cookbook/security/voters.rst

```diff
@@ -226,7 +226,7 @@ from the authorization checker is called.
 
             $authChecker = $this->get('security.authorization_checker');
 
-            $this->denyAccessUnlessGranted('view', $post, 'Unauthorized access!');
+            $authChecker->denyAccessUnlessGranted('view', $post, 'Unauthorized access!');
```

Comment:
My bad. But I still don't understand why is service instantiated in this example and then not used.